CryptoExchange.Net/Interfaces/ISymbolOrderBook.cs

```diff
@@ -23,6 +23,10 @@ public interface ISymbolOrderBook
         /// The symbol of the order book
         /// </summary>
         string Symbol { get; }
+        /// <summary>
+        /// price tick, e.g. for BTCUSDT pair at most exchanges it would be 0.01
+        /// </summary>
+        decimal? TickSize { get; }
```

Comment:
Why should this be part of the order book?

Reply:
it can be useful for making extensions for ISymbolOrderbook. for example, finding optimal price for placing order one tick better

Reply:
It feels like to me that is not really related to the order book though, so I would just keep that at the ICommonSymbol